src/entities/index.ts

```diff
@@ -24,3 +24,5 @@ export * from './types';
 export * from './utils';
 export * from './swap';
 export * from './swap/swaps/v2/auraBalSwaps';
+export * from './addLiquidityBoosted';
+export * from './removeLiquidityBoostedV3';
```

Comment:
Noticed one has V3 and one doesn't, lets use a consistent pattern, I'd say without V3?

Reply:
Done. Renamed the folders to not have v3 anymore. For the classnames I found it easier to leave them with V3. Can also change. What do you think?